### DISCUSSION TITLE
missing likehood and iternations in HDF5 file

### DISCUSSION THREAD

**Discussion by Z-Qing** [jungmannlab/picasso#290] [Q&A] [answered]
Hi, all. The doc about localization HDF5 files mentions two parameters the log-likelihood of the fit and the number of iterations of the fit procedure which I don't have. I guess they are deprecated. I want to ask which version of picasso: Localize still have that two columns? Is there a way I can export the two parameters in the latest version? Thanks.

**Comment by rafalkowalewski1** [comment]
Hi, these should not be changed in any picasso version. However, in the recent update (0.5), least squares has been changed to the default localizing method in the GUI version of localize. You simply need to change the fit method in the parameters windows.

**Reply by Z-Qing** [reply]
Hi. Thanks for the reply. I tried all three fitting methods in the GUI. But none of them provides the two columns.
I am using picasso 0.4.5 and windows 11.
![Screenshot 2022-11-21 103112](https://user-images.githubusercontent.com/103646272/203028718-7993e849-7f22-43cb-a140-e129d3303012.png)
![Screenshot 2022-11-21 103149](https://user-images.githubusercontent.com/103646272/203028747-2fa235b8-c0fd-437b-9b0b-3baf18fc14f7.png)

**Reply by Z-Qing** [reply]
Btw, the two columns show up when I use the CMD instead of GUI.

**Comment by rafalkowalewski1** [comment, marked as answer]
Thanks for the explanation. I'm sorry I misunderstood you in the beginning, I was worried that you cannot specify these parameters in the localize window. 

Picasso 0.2.1 should provide these columns: https://github.com/jungmannlab/picasso/releases/tag/v0.2.1
However, the release of picasso 0.2.4 says that MLE stability was improved, so using the versions before that may be a bit risky. 

I will consider saving these two columns in the future releases of picasso.

I hope this helps, let me know if you have further questions.

**Reply by Z-Qing** [reply]
Thanks for the answer. I will use the CMD of current version. I think it's a goood idea to have the parameters that describe the quality of fitting no matter which algo is used. Thanks for your help, again.
